app/controllers/orders/active_items.js

```diff
@@ -18,12 +27,51 @@ export default detail.extend({
     "sortProperties"
   ),
 
+  setDefaultSortingColumn: Ember.computed("dropDownItems", {
+    get(key) {
+      return ORDER_SORTING_OPTIONS[0];
+    },
+    set(key, value) {
+      return value;
+    }
+  }),
+
+  getStates() {
+    const utilities = this.get("utilityMethods");
+    const state = utilities.stringifyArray(this.getFilteredStates());
+    return {
+      state
+    };
+  },
+
+  getFilteredStates() {
+    return _.filter(this.get("states"), ["enabled", true]).map(
+      ({ state }) => state
+    );
+  },
+
+  stateList: Ember.observer("states.@each.enabled", function() {
+    this.reloadResults();
+    this.set("filteredStates", this.getFilteredStates());
+  }),
+
+  getSortQuery() {
+    const sortOptions = this.get("setDefaultSortingColumn");
+    return {
+      sort_column: sortOptions["column_alias"],
+      is_desc: sortOptions["is_desc"]
```

Comment:
```suggestion
      is_desc: /desc/.test(sortOptions["sort"])
```

you can also make it a computed property

```
sortQuery: Ember.computed('sortOptions', function () {
  const { column_alias, sort } = this.get("sortOptions");
  return {
    sort_column: column_alias,
    is_desc: /desc/i.test(sort)
  }
})
```